backend/src/main/scala/bloop/CompilerCache.scala

```diff
@@ -327,4 +327,17 @@ final class CompilerCache(
       }
     }
   }
+
+  /**
+   * Equivalent to `path1.isSameFile(path2)`, but will log exceptions and return `false`.
+   */
+  private def isSameFile(logger: Logger, path1: AbsolutePath, path2: AbsolutePath): Boolean = {
```

Comment:
What about changing the name of this method and make it more specific to java path? We can then provide a custom error in case java home doesn’t exist

Reply:
@Duhemm Feel free to merge when you address this!

Reply:
Done!